guide/daily-spending-wallet/receiving.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: guide
+title: Receiving bitcoin
+description: How receiving works for a daily spending bitcoin wallet.
+nav_order: 2
```

Comment:
I think this page should go after "Requesting bitcoin". Flows better, especially as the content begins with "Once a user has shared a payment request", which is explained on the requesting page.

Reply:
Completely agree, I was going to bring this up and do it in a separate PR. I think the below order makes the most logical sense as its the order of operations the user will likely (and what we recommend) will take.

First use -> Requesting -> Receiving -> Backup & recovery -> sending bitcoin

Thoughts @sbddesign ?